helpers/test/focus.test.js

```diff
@@ -358,4 +376,32 @@ describe('focus', () => {
 
 	});
 
+	describe('tryApplyFocus', () => {
+
+		it('returns true on single focusable element', async() => {
+			const elem = await fixture(focusableFixture);
+			expect(tryApplyFocus(elem.querySelector('button'))).to.be.true;
+		});
+
+		it('returns false on non-existent element', async() => {
+			const elem = await fixture(focusableFixture);
+			expect(tryApplyFocus(elem.querySelector('#nonExistentDiv'))).to.be.false;
+		});
+
+		it('returns true on element with focusable parent', async() => {
+			const elem = await fixture(focusableAncestorFixture);
```

Comment:
Nit: one thing you could add to these tests is to not only check for the `true`/`false` return value, but also ensure that it's actually focusing on the element you expect. To do that, you could give them `id`s and ensure that `elem.id` matches what you're expecting.